README.md

```diff
@@ -42,14 +55,66 @@ var output = postcss([
 console.log(output);
 ```
 
-[*For more general PostCSS usage, see this section*](https://github.com/postcss/postcss#usage)
+# Syntax
+
+### Defining Custom Properties: the `--*` family of properties
```

Comment:
The rework done in this section sounds worse than current.

Let's keep the current and just add a `.foo` example to the existing code fence.

Let's also adjust the heading:

```
### Defining Variables/Custom Properties, `--*`
```